packages/perseus/src/styles/perseus-renderer.less

```diff
@@ -1004,5 +1004,5 @@
     }
 }
 
-@import (inline) "../../../../node_modules/@khanacademy/math-input/build/math-input.css";
+@import (inline) "../../../../node_modules/@khanacademy/math-input/dist/index.css";
```

Comment:
This requires that `math-input` is built before `perseus` is built.  Is this the order that rollup is building things in?

Reply:
Actually, we had no guarantees on build order. But #178 fixes that! :)